tests/flows/test_sklearn.py

```diff
@@ -0,0 +1,459 @@
+        self.assertEqual(serialization.name,
+                         'sklearn.pipeline.FeatureUnion('
+                         'sklearn.preprocessing.data.OneHotEncoder)')
+        new_model = flow_to_sklearn(serialization)
+        self.assertEqual(type(new_model), type(fu))
+        self.assertIsNot(new_model, fu)
+        self.assertIs(new_model.transformer_list[1][1], None)
+
+    def test_serialize_complex_flow(self):
+        ohe = sklearn.preprocessing.OneHotEncoder(categorical_features=[0])
+        scaler = sklearn.preprocessing.StandardScaler(with_mean=False)
+        boosting = sklearn.ensemble.AdaBoostClassifier(
+            base_estimator=sklearn.tree.DecisionTreeClassifier())
+        model = sklearn.pipeline.Pipeline(steps=(
+            ('ohe', ohe), ('scaler', scaler), ('boosting', boosting)))
+        parameter_grid = {'n_estimators': [1, 5, 10, 100],
+                          'learning_rate': scipy.stats.uniform(0.01, 0.99),
+                          'base_estimator__max_depth': scipy.stats.randint(1,
+                                                                           10)}
+        cv = sklearn.model_selection.StratifiedKFold(n_splits=5, shuffle=True)
+        rs = sklearn.model_selection.RandomizedSearchCV(
+            estimator=model, param_distributions=parameter_grid, cv=cv)
+        serialized = sklearn_to_flow(rs)
+
+        fixture_name = 'sklearn.model_selection._search.RandomizedSearchCV(' \
+                       'estimator=sklearn.pipeline.Pipeline(' \
+                       'sklearn.preprocessing.data.OneHotEncoder,' \
+                       'sklearn.preprocessing.data.StandardScaler,' \
+                       'sklearn.ensemble.weight_boosting.AdaBoostClassifier(' \
+                       'base_estimator=sklearn.tree.tree.DecisionTreeClassifier)))'
+        self.assertEqual(serialized.name, fixture_name)
+
+        # now do deserialization
+        deserialized = flow_to_sklearn(serialized)
+
+        serialized2 = sklearn_to_flow(deserialized)
+        self.assertNotEqual(rs, deserialized)
+        self.assertEqual(serialized, serialized2)
+
+    def test_serialize_type(self):
+        supported_types = [float, np.float, np.float32, np.float64,
+                           int, np.int, np.int32, np.int64]
+
+        for supported_type in supported_types:
+            serialized = sklearn_to_flow(supported_type)
+            deserialized = flow_to_sklearn(serialized)
+            self.assertEqual(deserialized, supported_type)
+
+    def test_serialize_rvs(self):
+        supported_rvs = [scipy.stats.norm(loc=1, scale=5),
+                         scipy.stats.expon(loc=1, scale=5),
+                         scipy.stats.randint(low=-3, high=15)]
+
+        for supported_rv in supported_rvs:
+            serialized = sklearn_to_flow(supported_rv)
+            deserialized = flow_to_sklearn(serialized)
+            self.assertEqual(type(deserialized.__dict__['dist']),
```

Comment:
That makes the code easier, thanks. It compares that the type of the distribution object is the same after serialization and deserialization.